pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/dashboard/DashboardLabelDto.java

```diff
@@ -0,0 +1,67 @@
+package org.broadinstitute.dsm.db.dto.dashboard;
+
+import java.util.Optional;
+
+import lombok.Data;
+import org.apache.commons.lang3.StringUtils;
+
+@Data
```

Comment:
what is this class intended for?  what is a "DashboardLabel"?  Please add comment